cobc/cobc.c

```diff
@@ -1316,6 +1316,61 @@ cobc_plex_strdup (const char *dupstr)
 	return p;
 }
 
+/* Return a newly allocated string with only the first len chars of the first argument */
+void *
+cobc_plex_strsub (const char *s, const int len)
+{
+	void	*p;
+	int	n;
+
+#ifdef	COB_TREE_DEBUG
```

Comment:
the `!s` checks here and in `cobc_plex_stradd` can be removed as we use the parameter(s) to call `strlen` - in the case of `NULL` we'll get a SIGSEGV that will also create a nice coredump or allows us to stop there